RecoTauTag/Configuration/python/HPSPFTaus_cff.py

```diff
@@ -374,74 +386,45 @@
         )
     )
 )
+def _modMapping(mapping, wpIn, wpOut, verIn="v1", verOut="v3_noeveto"):
+    for m in mapping:
+        m.cut = m.cut.value().replace(verIn, verOut)
+        m.cut = m.cut.value().replace(wpIn, wpOut)
+run2_miniAOD_UL.toModify(
+    hpsPFTauDiscriminationByMVA6VLooseElectronRejection.mapping,
+    func = lambda mapping: _modMapping(mapping, "WPEff99", "WPeff98")
+)
 ## ByMVA6LooseElectronRejection
 hpsPFTauDiscriminationByMVA6LooseElectronRejection = copy.deepcopy(hpsPFTauDiscriminationByMVA6VLooseElectronRejection)
 for m in hpsPFTauDiscriminationByMVA6LooseElectronRejection.mapping:
     m.cut = m.cut.value().replace("WPEff99", "WPEff96")
+run2_miniAOD_UL.toModify(
+    hpsPFTauDiscriminationByMVA6LooseElectronRejection.mapping,
+    func = lambda mapping: _modMapping(mapping, "WPeff98", "WPeff90")
```

Comment:
shouldn't this be changing values of ```hpsPFTauDiscriminationByMVA6LooseElectronRejection.mapping``` defined above, which has WPEff96?
```suggestion
    func = lambda mapping: _modMapping(mapping, "WPeff96", "WPeff90")
```

Reply:
ah, actually the capitalization is also different (someone was creative when making the payloads)
```suggestion
    func = lambda mapping: _modMapping(mapping, "WPEff96", "WPeff90")
```

Reply:
No, actually current version is correct. It is as cloned module (by VLoose) is changed by the UL modifier before it is cloned, i.e. 1st set of mapping changes of type "WPEff99->WPEffXX does nothing with the UL modifier as WPEff99 is already replaced by WPeff98. 
And yes, it is a pity that capitalization in payloads' names changes.

Reply:
nevermind, I'm wrong, because the ```m.cut = m.cut.value().replace("WPEff99", "WPEff96")``` is inactive because the copy/clone above starts from ```hpsPFTauDiscriminationByMVA6VLooseElectronRejection``` already under ```run2_miniAOD_UL```

Reply:
Yes, exactly.